app/templates/main/manageServiceLearningFaculty.html

```diff
@@ -50,23 +50,25 @@ <h5 class="ms-3">There are no unapproved courses for {{ term.description }}.</h5
           <table class="table table-striped border">
             <thead>
               <tr>
-                <th scope="col">Courses</th>
-                <th scope="col">Faculty</th>
-                <th scope="col">Status</th>
-                <th scope="col"></th>
-                <th scope="col">Edit</th>
+                <th class="col-md-1" scope="col">Courses</th>
+                <th class="col-md-1" scope="col">Faculty</th>
+                <th class="col-md-1" scope="col">Status</th>
+                <th class="col-md-1" scope="col">Action</th>
               </tr>
             </thead>
             <tbody>
               {% for course in unapprovedCourses %}
               <tr>
                 <td>{{course.courseName}}</td>
+                <td>{{ course.instructors }}</td>
+                <td>{{course.status.status}}</td>
                 <td>
-                  {{ course.instructors }} <br>
+                  <select class="form-select courseData" id="{{course.id}}" onchange='courseAction(this)'>
```

Comment:
is there a `courseData` class? I don't see where this is ever defined or referenced